README.md

```diff
@@ -1 +1,45 @@
-# Final-Javascript-capstone
\ No newline at end of file
+# Javascript-capstone
+
+Our JavaScript capstone project is about building a website that shows the most popular movies from an API with the ability to comment and make reservations on each movie. The webapp has 3 user interfaces:
+
```

Comment:
- [ ] As mentioned by the previous reviewer, there is a need for a `Getting tarted` section of the README in which you should list step-by-step instructions on how to get started with your App locally on one's computer.
  
  Please add a section called `Getting started`, then in this section list steps on how can get set up with your project locally on a computer.